spec/intl.html

```diff
@@ -93,32 +148,46 @@ <h1>
         1. If _ianaTimeZone_ is *"Etc/UTC"* or *"Etc/GMT"*, return *"UTC"*.
         1. Return _ianaTimeZone_.
       </emu-alg>
-
-      <ins class="block">
-        <p>This definition supersedes the definition provided in <emu-xref href="#sec-canonicalizetimezonename"></emu-xref>.</p>
-      </ins>
     </emu-clause>
+    </del>
 
     <ins class="block">
-      <emu-clause id="sup-availabletimezones" type="implementation-defined abstract operation">
-        <h1>
-          AvailableTimeZones (
-          ): a List of Strings
-        </h1>
+      <emu-clause id="sup-availabletimezoneidentifiers" oldids="sec-availabletimezones" type="implementation-defined abstract operation">
+        <h1>AvailableTimeZoneIdentifiers ( ): a List of Time Zone Identifier Records</h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>The returned List is a sorted List of supported Zone and Link names in the IANA Time Zone Database.</dd>
+          <dd>
+            Its result describes all available time zone identifiers in this implementation, as well as the primary time zone identifier corresponding to each available identifier.
+            The List is sorted by [[Identifier]] of each Time Zone Identifier Record.
+          </dd>
           <dt>redefinition</dt>
           <dd>true</dd>
         </dl>
         <emu-alg>
-          1. Let _timeZones_ be a List containing the String value of each Zone or Link name in the IANA Time Zone Database that is supported by the implementation.
-          1. Assert: _timeZones_ contains *"UTC"*.
-          1. Assert: _timeZones_ does not contain any element that does not identify a Zone or Link name in the IANA Time Zone Database.
-          1. Return SortStringListByCodeUnit(_timeZones_).
+          1. Let _identifiers_ be a List containing the String value of each Zone or Link name in the IANA Time Zone Database that is supported by the implementation.
+          1. Assert: No element of _identifiers_ is an ASCII-case-insensitive match for any other element.
+          1. Assert: No element of _identifiers_ does not identify a Zone or Link name in the IANA Time Zone Database.
```

Comment:
Phrasing suggestion, double negatives can be harder to read:
```suggestion
          1. Assert: Every element of _identifiers_ identifies a Zone or Link name in the IANA Time Zone Database.
```